src/main/presenter/mcpPresenter/inMemoryServers/appleServer.ts

```
@@ -0,0 +1,1770 @@
+            ]
+          }
+
+        case 'search':
+          if (!parsedArgs.searchText) {
+            throw new Error('Search text is required for search operation')
+          }
+          const searchResults = await NotesUtils.findNote(parsedArgs.searchText)
+          return {
+            content: [
+              {
+                type: 'text' as const,
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)